projects/admin/src/app/record/custom-editor/libraries/pipe/notificationType.pipe.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'notificationType'
+})
+export class NotificationTypePipe implements PipeTransform {
+
+  private _notification = {
+    patron: [
+      'availability',
+      'due_soon',
+      'overdue',
+      'recall'
+    ],
+    library: [
+      'booking',
+      'request',
+      'transit_notice'
+    ]
+  };
```

Comment:
For me, these values should be deported into a 'Notification class model' outside of the pipe.
In few months, it shouldn't be easy to remember than this config is into this Pipe.